packages/cli/src/commands/verify.ts

```diff
@@ -0,0 +1,51 @@
+import { Args, Command, Flags } from '@oclif/core';
+import fs from 'fs/promises';
+import { sigstore } from 'sigstore';
+
+export default class Verify extends Command {
+  static override description = 'describe the command here';
+  static override examples = [
+    '<%= config.bin %> <%= command.id %> ./bundle.sigstore',
+  ];
+  static override enableJsonFlag = true;
+
+  static override flags = {
+    'tlog-threshold': Flags.integer({
```

Comment:
Presuming you punted on the other options until we decide exactly how to expose these like?
- certificateIssuer?: string;
- certificateIdentityEmail?: string;
- certificateIdentityURI?: string;
- certificateOIDs?: Record<string, string>;
- tufMirrorURL?: string;
- tufRootPath?: string;
- tufCachePath?: string;
- retry
- timeout

Reply:
Yeah, will come back and address all of those later. I feel like the `certificate*` options need some refactoring still and the rest of those probably need to be global options (shared across all the CLI commands) but I think that requires that I extract a shared bass class.